docs/languages/csharp.md

```diff
@@ -10,7 +10,7 @@ MetaDescription: Find out how to get the best out of Visual Studio Code and C#.
 
 # Working with C&#35;
 
-The C# support in VS Code is optimized for cross-platform .NET Core development (see [working with .NET Core and VS Code](/docs/runtimes/dotnet.md) for another relevant article).  Our focus with VS Code is to be a great editor for cross-platform C# development.  For instance, many Unity game developers enjoy using VS Code in place of the MonoDevelop IDE.
+The C# support in VS Code is optimized for cross-platform .NET Core development (see [working with .NET Core and VS Code](/docs/runtimes/dotnet.md) for another relevant article).  Our focus with VS Code is to be a great editor for cross-platform C# development.  For instance, many [Unity](https://code.visualstudio.com/docs/runtimes/unity) game developers enjoy using VS Code in place of the MonoDevelop IDE.

```

Comment:
For links within the code.visualstudio.com site, you can use relative links. In this case, use /docs/runtimes/unity.md.
Using the .md file extension enables GitHub navigation as some of our users (especially for our vnext preview branch) read on GitHub.